.env

```
@@ -0,0 +1,14 @@
+PORT=3001
+NODE_ENV=development
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NODE_ENV key should go before the PORT key

(UnorderedKey)